### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by purplefox**
Someone please review @vietj @cescoffier @pmlopes

### CODE REVIEW

**Comment by cescoffier** [comment]
Will do.

File: src/main/asciidoc/index.adoc [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+It is quite common in particular on OSX machines for multicast to be disabled by default. Please google for
+information on how to enable this.
+
+=== Using wrong network interface
+
+If you have more than one network interface on your machine (and this can also be the case if you are running
+VPN software on your machine), then Hazelcast may be using the wrong one.
+
+To tell Hazelcast to use a specific interface you can provide the IP address of the interface in the `interfaces`
+element of the configuration. Make sure you set the `enabled` attribute to `true`.
```
Comment (cescoffier) [inline src/main/asciidoc/index.adoc]: What about adding an example here such as:

        <interfaces enabled="true">
            <interface>192.168.1.20</interface>
        </interfaces>

File: src/main/asciidoc/index.adoc [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+
+=== Using a VPN
+
+This is a variation of the above case. VPN software often works by creating a virtual network interface which often
+doesn't support multicast. If you have a VPN running and you do not specify the correct interface to use in both the
+hazelcast configuration and to Vert.x then the VPN interface may be chosen instead of the correct interface.
+
+So, if you have a VPN running you may have to configure both the Hazelcast and Vert.x to use the correct interface as
+described in the previous section.
+
```
Comment (cescoffier) [inline src/main/asciidoc/index.adoc]: Maybe mention that they may have to use tcp / unicast here

**Comment by cescoffier** [comment]
I've made two small comments (obviously on the adco file and not on the pacakge-info.java file....). 

Everything else is fine. So you can merge (I don't have the right to merge on this repo).

**Comment by purplefox** [comment]
Thanks for the comments clement, I have added a few more bits as you suggested.

**Pull request merged**